apps/dashboard/src/lib/utils/translations/fr.json

```
@@ -862,6 +862,10 @@
           "description_live": "Les leçons et les exercices verrouillés restent cachés aux étudiants jusqu'à ce que vous les déverrouilliez, ce qui est utile si vous publiez du contenu semaine après semaine.",
           "unlock_all": "Débloquez tout"
         },
+        "certificate_deadline_required": "Définir d'abord la date limite du certificat",
```

Biome (2.5.7)

[error] The key certificate_deadline_required was already declared.

(lint/suspicious/noDuplicateObjectKeys)